internal/job/starter.go

```diff
@@ -79,6 +80,7 @@ type RDCStarterOptions struct {
 	TestFramework string            `json:"test_framework"`
 	AppID         string            `json:"app_id"`
 	TestAppID     string            `json:"test_app_id"`
+	OtherApps     []string          `json:"other_apps"`
```

Comment:
Can you please delete this entire struct? It's dead code and nothing is using it.

Reply:
Oh ok!